src/v/kafka/client/test/fixture.h

```diff
@@ -28,6 +28,7 @@ class kafka_client_fixture : public redpanda_thread_fixture {
         }).get0();
         app.initialize(proxy_config(), proxy_client_config());
         app.check_environment();
+        app.clear_migrators();
         app.wire_up_and_start(*app_signal);
```

Comment:
I think you want a `true` for the optional test_mode argument to `wire_up_and start` here

Reply:
ah right!